overlord/fdestate/fdemgr.go

```diff
@@ -87,6 +88,23 @@ func Manager(st *state.State, runner *state.TaskRunner) (*FDEManager, error) {
 	defer st.Unlock()
 	st.Cache(fdeMgrKey{}, m)
 
+	snapstate.RegisterAffectedSnapsByKind("efi-secureboot-db-update", dbxUpdateAffectedSnaps)
```

Comment:
we still need to test this with something like doing a snapstate.Update on kernel while the fw update change is around?

Reply:
added a unit test